src/navigate-pop/using-workspaces.md

```diff
@@ -0,0 +1,48 @@
+# Workspaces
+
+Workspaces serve to reduce window clutter, especially on smaller displays. 
+
+---
+
+## Using Workspaces
+
+Show and access workspaces by clicking on the workspaces icon in the dock, clicking `Workspaces` in the top left corner of your screen, or by pressing `SUPER` + `D`.
+
+The open windows for a given workspace will display when that workspace is selected.
+
+<video autoplay loop>
+        <source src="/images/workspaces/show-workspaces.webm" />
```

Comment:
You had five windows minimized at the start of this animation? I think this could be confusing for someone just learning about Workspaces for the first time-- the workspace preview is empty because all of the windows were minimized, so it's not clear that preview is showing the current workspace with those windows. Having one or two minimized windows to demonstrate that they show up in the Workspaces menu is a good idea, but I would at least have some of the windows un-minimized before opening the menu.